PSSwagger/PSSwagger.psm1

```diff
@@ -800,6 +822,70 @@ function New-PSSwaggerModule {
             -PSHeaderComment $PSHeaderComment
     }
 
+    if ($CopyUtilityModuleToOutput) {
+        Write-Verbose -Message $LocalizedData.CopyingUtilityModule
+        $utilityModuleInfo = Get-Module PSSwaggerUtility
+        $existingPath = (Join-Path -Path $outputDirectory -ChildPath PSSwaggerUtility)
+        Write-Warning -Message ($LocalizedData.ReSignUtilityModuleWarning -f $existingPath)
+        if (Test-Path -Path $existingPath) {
+            $null = Remove-Item -Path $existingPath -Recurse -Force
+        }
+
+        $null = New-Item -Path $existingPath -ItemType Directory -Force
+        foreach ($item in Get-ChildItem -Path $utilityModuleInfo.ModuleBase) {
+            if ($item.Name -eq 'PSSwaggerClientTracing.psm1') {
+                Write-Warning -Message $LocalizedData.TracingDisabled
+                Copy-Item -Path (Join-Path -Path $PSScriptRoot -ChildPath 'PSSwaggerClientTracing_Dummy.psm1') -Destination (Join-Path -Path $existingPath -ChildPath $item.Name)
```

Comment:
>PSSwaggerClientTracing_Dummy.psm1 [](start = 75, length = 33)

Where is this file PSSwaggerClientTracing_Dummy.psm1 getting created?

Reply:
Forgot to add, will add